httpserver/src/main/kotlin/tbdex/sdk/httpserver/TbdexHttpServer.kt

```diff
@@ -99,33 +98,34 @@ class TbdexHttpServer(val config: TbdexHttpServerConfig) {
         call.respond(
           HttpStatusCode.OK, "Please use the tbdex protocol " +
           "via a suitable library to communicate with this server: " +
-          "https://github.com/TBD54566975/tbdex-protocol"
+          "https://github.com/TBD54566975/tbdex"
         )
       }
 
+      // todo i don't want do unchecked cast here
       route("/exchanges") {
         post {
           createExchange(
             call = call,
             offeringsApi = offeringsApi,
             exchangesApi = exchangesApi,
-            callback = submitCallbacks.getOrDefault("rfq", null)
+            callback = callbacks.getOrDefault("rfq", null) as CreateExchangeCallback?
```

Comment:
Rather than having an array of callbacks (since we only have one implementation of each) should we just have them each typed out as their own var?

Reply:
see https://github.com/TBD54566975/tbdex-kt/pull/218#discussion_r1544832930